### ISSUE TITLE
Can you change Jabber to XMPP?

### ISSUE THREAD

**Issue by Neustradamus** [jenkinsci/jabber-server-plugin#5]
Can you change Jabber to XMPP?
- https://github.com/jenkinsci/jabber-plugin
- https://github.com/jenkinsci/jabber-server-plugin

It is XMPP since 2004.

**Comment by enexusde** [comment]
I totally agree and I am unhappy with the name jabber too because there are some differences between jabber and XMPP ( https://tools.ietf.org/html/rfc3920#appendix-D ).  In the real world it is NOT jabber and the naming jabber-server-plugin is wrong BUT it is XMPP and the naming xmpp-server-plugin is correct! The decision to name the Plug-in jabber-server instead of xmpp-server was made 6 years ago at a time I had neither experiences with XMPP nor Jabber. 

6 Years ago there was only one plugin for xmpp called jabber-plugin.

I will check if I am be able to continue the development process that worked 6 years ago and try to release the plug-in in a different name.

**Comment by Neustradamus** [comment]
@enexusde: Thanks for your reply!
It is easy to rename a project, not create a new ^^

**Comment by Neustradamus** [comment]
Any news?

**Comment by Neustradamus** [comment]
Any news?

It is easy to rename two repositories...

**Comment by enexusde** [comment]
Not yet, why is it important for you? Do you need the name? Would you like to takeover full responsibility for this project?
